test/runtests.jl

```diff
@@ -94,7 +94,7 @@ end
     @test_throws ArgumentError time_from_index(200, 0)
     @test time_from_index(100, 1) == Nanosecond(0)
     @test time_from_index(100, 301:600) == TimeSpan(Second(3), Second(6))
-    @test time_from_index(100, 101:101) == TimeSpan(Second(1))
+    @test time_from_index(100, 101:101) == TimeSpan(Second(1), Nanosecond(1010000000))
```

Comment:
This also matches AlignedSpans: https://beacon-biosignals.github.io/AlignedSpans.jl/dev/#Discrete-Continuous

I.e. index 101 is associated to the whole 1/fs span from the time at which 101 occurred (inclusive) until the time of the next sample (exclusive)